pkg/gameserverallocations/allocation_cache_test.go

```diff
@@ -500,13 +480,10 @@ func TestAllocationCacheCompareGameServers(t *testing.T) {
 	}
 }
 
-func TestAllocatorRunCacheSyncFeatureStateAllocationFilter(t *testing.T) {
+func TestAllocatorRunCacheSync(t *testing.T) {
 	t.Parallel()
 
 	// TODO(markmandel): When this feature gets promoted to stable, replace test TestAllocatorRunCacheSync below with this test.
```

Comment:
We can drop the TODO as well here please, since we did the replacement 😄

Reply:
done!